### DISCUSSION TITLE
Is there integration for VSCode?

### DISCUSSION THREAD

**Discussion by tomereli** [johnthagen/python-blueprint#162] [Q&A]
The blueprint is amazing, how do I make it play nice in VSCode?

**Comment by johnthagen** [comment]
@tomereli Glad you like `python-blueprint`! In general, since I don't use VS Code as my full time editor I don't document any specific tricks needed for it.

That being said, since this project isn't doing anything that is really non-standard, I wouldn't expect users to have many issues using this project style with VS Code.
